src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
+
+import com.revrobotics.ColorMatch;
+import com.revrobotics.ColorMatchResult;
+import com.revrobotics.ColorSensorV3;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj.util.Color;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.robot.Constants;
+import frc.robot.RobotMap.mapIntake;
+
+public class Intake extends SubsystemBase {
+
+  ColorSensorV3 intakeColorSensor;
+  ColorMatch colorMatcher;
+  Color coneColor = new Color(0.34509, 0.51764, 0.13333);
+  Color cubeColor = new Color(0.22745, 0.39607, 0.37254);
```

Comment:
We probably want to store this in constants or have each of the values tunable with robot preference